app/objective/post.controller.js

```diff
@@ -0,0 +1,59 @@
+const { body } = require('express-validator')
+const { logger } = require('../../common/logging/logger')
+const { setNewSentencePlanObjective, updateSentencePlanObjective } = require('../../common/data/sentencePlanningApi')
+const { getObjective } = require('./get.controller')
+const { countWords, removeUrlLevels } = require('../../common/utils/util')
+
+const wordsAllowed = 50
+
+const validationRules = () => {
+  return [
+    body('objective')
+      .isLength({ min: 1 })
+      .withMessage('Describe the objective'),
+    body('objective')
+      .custom(value => {
+        return countWords(value) <= wordsAllowed
+      })
+      .withMessage('Objective description must be 50 words or fewer'),
+  ]
+}
+
+const postObjective = async (req, res) => {
+  const {
+    path,
+    errors,
+    body: { objective: objectiveDescription },
+    params: { planId, objectiveId },
+    session: { 'x-auth-token': token },
+  } = req
+
+  const backurl = removeUrlLevels(path, 2)
```

Comment:
Move this line to after the `if (errors)...` block to save it being evaluated pointlessly

Reply:
Actually just move it to line 50 where it is used.